app/controllers/mfa_confirmation_controller.rb

```diff
@@ -1,8 +1,7 @@
 class MfaConfirmationController < ApplicationController
   before_action :confirm_two_factor_authenticated
 
-  def new
-  end
+  def new; end
```

Comment:
since rails doesn't need us to define these empty controller methods, thoughts on deleting altogether? I know some ppl like to keep them for clarity.